torch/csrc/jit/runtime/interpreter.cpp

```diff
@@ -1611,6 +1646,27 @@ struct InterpreterStateImpl : c10::intrusive_ptr_target {
   }
 };
 
+std::vector<FileLineFunc> currentCallstack() {
```

Comment:
This couples the profilers FileLineFunc struct to the interpreter in a way that doesn't seem necessary. FileLineFunc seems like a duplicate of StackEntry here. The more flexible format for the interpreter to return is the StackEntry list. The interpreter should just return this list, and if a client like the profiler wants to translated into a simpler form like below it should do it there.